website/content/docs/security/acl/acl-system.mdx

```diff
@@ -179,26 +215,66 @@ of the following elements:
 - **Service Identity Set** - The list of service identities that are applicable for the role.
 - **Namespace** <EnterpriseAlert inline /> - The namespace this policy resides within. (Added in Consul Enterprise 1.7.0)
 
--> **Consul Enterprise Namespacing** - Roles may only link to policies defined in the same namespace as the role itself.
+-> **Linking Roles to Policies in Consul Enterprise** - Roles can only be linked to policies that are defined in the same namespace or admin partition. 
 
 ### ACL Tokens
 
-ACL tokens are used to determine if the caller is authorized to perform an action. An ACL token is composed of the following
-elements:
+Consul uses ACL tokens to determine if the caller is authorized to perform an action. An ACL token is composed of several attributes that you can specify when creating the token. Refer to the [ACL token command line](https://www.consul.io/commands/acl/token) documentation or [ACL token API](/api-docs/acl/tokens) documentation for additional information on how to create policies.:
```

Comment:
There are some references to policies below that I think should instead be to tokens:
```suggestion
Consul uses ACL tokens to determine if the caller is authorized to perform an action. An ACL token is composed of several attributes that you can specify when creating the token. Refer to the [ACL token command line](https://www.consul.io/commands/acl/token) documentation or [ACL token API](/api-docs/acl/tokens) documentation for additional information on how to create tokens.:
```